Samples/SpawnOnEntity.cs

```diff
@@ -13,22 +13,25 @@ public class SpawnOnEntity : MonoBehaviour
     /// <summary>
     /// GameObject to spawn representing entities.
     /// </summary>
-    public GameObject SpawnPrefab = null;
+    [Tooltip("Spawnable 3D object")]
```

Comment:
Trenger ikke være 3D, må bare være et GameObject/ha Transform komponenten
```suggestion
    [Tooltip("Spawnable prefab")]
```